### PR TITLE
Adjust the dimensions of `top-bar` and `annotator-toolbar` on mobile

### PR DESCRIPTION
**Pull request by esanzgar**
On mobile screen sizes, buttons get a minimum recommended size (following Apple's Human Interface Guidelines). To accommodate for that, `top-bar` height and `annotator-toobar` width must be adjusted accordingly.

Final result:
image

### CODE REVIEW

**Comment by esanzgar** [comment]
I did not tested it on real mobile device, but on device emulator.

**Review by lyzadanger — CHANGES_REQUESTED** [review]
Hi! This is moving in a good direction, thank you!

Here is a screenshot from my iPhone (8):

image

Note that the left side and border of the buttons are cut off. This is going to be tricky to account for, as we simply don't quite have enough room for the 44px-wide button targets to the left of the sidebar content. But perhaps you can brainstorm something? It might require positioning the two lower buttons and nudging them to the right by a few pixels. It might need more cleverness than that. I'm not sure!

**Comment by esanzgar** [comment]
@lyzadanger, unfortunately moving the two lower buttons to the right by a few pixel makes it to overlap with other content:

image

**Comment by esanzgar** [comment]
I decided to make the sidebar a little bit smaller:
image

**Review by lyzadanger — CHANGES_REQUESTED** [review]
Hi @esanzgar 

Unfortunately, you've inherited a larger design issue here related to proportional dimensions of the sidebar on narrow-width device viewports. A larger restructure is needed her in future to have a more truly adaptive layout for various viewports/constraints.

For the purposes of making the buttons look and feel right on viewports that are in the iPhone-ish smartphone range, I think we may need an interim compromise.

What I'd suggest is hand-sizing these buttons/targets on narrow viewports to be oh, say, 38px wide or perhaps 40px—whatever the maximum width that we can fit on-screen for viewports in the target problem area (iPhone 8-ish). I totally acknowledge that this sucks—it's smaller than HIG suggest, but still larger than the button targets would be with no intervention.

I believe what we need to do in the future is refactor these narrow-viewport sidebar affordances—we can't be successful trying to position/fit/size buttons that have absolute dimensions but also have a dependency on a relative-unit-sized layout.

I'd like a sanity check from @robertknight on my general perspective here.

File: src/styles/annotator/annotator.scss [hunk 1]
```diff
@@ -107,8 +107,8 @@
 
 @media screen and (min-width: 15em) {
   .annotator-frame {
-    width: 90%;
-    margin-left: -90%;
+    width: 84%;
```
[the remark this answers is not in the document]
Comment (esanzgar) [reply src/styles/annotator/annotator.scss:110]: I have checked on the range of screen width (<= 380px and >=240px) that displayed this issue and this solution seems to resolve the cutting-off of the buttons, while still adhering to Apple's HIG.

[the remark this answers is not in the document]
Comment (lyzadanger) [reply src/styles/annotator/annotator.scss:110]: Sorry, I should have also mentioned the other pushback we're likely to encounter: we need to take significant pains not to reduce the width of the sidebar, as that is the content that the user needs to interact with. That real estate is very precious.

[the remark this answers is not in the document]
Comment (esanzgar) [reply src/styles/annotator/annotator.scss:110]: @lyzadanger, please let me know if you want me to implement your proposed solution (resizing the buttons on narrow-width screens) instead.

Previous conversation: https://hypothes-is.slack.com/archives/C1M8NH76X/p1605782090037200

[the remark this answers is not in the document]
Comment (lyzadanger) [reply src/styles/annotator/annotator.scss:110]: Yes, please. We need to fix the bigger picture here—what I'm proposing is interim—but we don't have the time to bite this off just now.

**Comment by esanzgar** [comment]
I implemented @lyzadanger's interim solution: reducing the size of the buttons in the `annotator-bar` for all mobile devices (even on large screen devices) without affecting the size of the sidebar.

The final size of the buttons are 22px. This is a comprise to get the bigger possible `annotator-bar` buttons without cropping them. This is a screenshoot on an 320px wide screen (iPhone):

image

On screens smaller than 320px wide the `annotator-bar` buttons get cropped a little. There are very few devices with screen narrower than 320px.

**Review by lyzadanger — APPROVED** [review]
Thanks for working through this and implementing an interim compromise.

**Pull request merged**